build.gradle

```diff
@@ -25,20 +28,33 @@ dependencies {
     implementation group: 'com.squareup.retrofit2', name: 'converter-gson', version: '2.6.2'
 
     implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.28'
-    testImplementation group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.28'
+    // testImplementation group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.28'
```

Comment:
Why commented out?

Reply:
At that time it was still a work in progress. It has been resolved with the latest work.